### PR TITLE
fix cli command user-project list

### PR DESCRIPTION
**Pull request by LuckySB**
fix https://github.com/python-gitlab/python-gitlab/issues/881

author of code fix @carleski
https://github.com/python-gitlab/python-gitlab/pull/669
But his request has not been accepted for a year and there is a need for rebase.
I made a new PR with the same fix

### CODE REVIEW

**Comment by max-wittig** [comment]
Hi @LuckySB. Thanks for the merge request. Could you please use conventional changelog for this merge request.

**Comment by LuckySB** [comment]
well done
all checks passed

/assign @max-wittig

File: gitlab/v4/objects.py [hunk 1]
```diff
@@ -278,8 +278,10 @@ def list(self, **kwargs):
             GitlabAuthenticationError: If authentication is not correct
             GitlabListError: If the server cannot perform the request
         """
-
-        path = "/users/%s/projects" % self._parent.id
+        if self._parent is not None:
```
Comment (max-wittig) [inline gitlab/v4/objects.py]:
```python
if self._parent:
```

Comment (LuckySB) [reply gitlab/v4/objects.py]: changed

**Pull request merged**